### PR TITLE
Fix hotbar light theme colors

### LINKED ISSUE #2786: Fix hotbar light theme colors
Colors for active cluster and empty hotbar cells are missing in light theme.

light theme hotbar

### PR DESCRIPTION
**Pull request by vshakirova**
Wdyt?
Screenshot from 2021-06-03 12-30-16

Suggestion: change `tableHeaderBackground` for themes consistency
photo_2021-06-03_12-29-18
photo_2021-06-03_12-29-21

Wdyt?

fixes: https://github.com/lensapp/lens/issues/2786

### CODE REVIEW

**Review by Nokel81 — DISMISSED** [review]

File: src/renderer/themes/lens-light.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "mainBackground": "#f1f1f1",
     "secondaryBackground": "#f2f3f5",
     "contentColor": "#ffffff",
-    "layoutBackground": "#e8e8e8",
+    "layoutBackground": "#d6d6d6",
```
Comment (aleksfront) [inline src/renderer/themes/lens-light.json]: Please do not change this one. It affects topbars across the dashboard.

File: src/renderer/themes/lens-light.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "buttonDisabledBackground": "#808080",
     "tableBgcStripe": "#f8f8f8",
     "tableBgcSelected": "#f4f5f5",
-    "tableHeaderBackground": "#f1f1f1",
+    "tableHeaderBackground": "#ffffff",
```
Comment (aleksfront) [inline src/renderer/themes/lens-light.json]: It affects general tables style. It was better with "grayed" table head.

Tables before the change:
gray table head

Tables after the change:
white table head

File: src/renderer/themes/lens-light.json [hunk 3]
```diff
@@ -104,7 +104,7 @@
     "drawerSubtitleBackground": "#f1f1f1",
     "drawerItemNameColor": "#727272",
     "drawerItemValueColor": "#555555",
-    "clusterMenuBackground": "#e8e8e8",
+    "clusterMenuBackground": "#c3c3c3",
```
Comment (aleksfront) [inline src/renderer/themes/lens-light.json]:
```suggestion
    "clusterMenuBackground": "#d7d8da",
```

**Review by aleksfront — CHANGES_REQUESTED** [review]
Along with other comments, please add new value for light theme
```
"clusterMenuCellBackground": "#bbbbbb"
```

for dark theme
```
"clusterMenuCellBackground": "#2e3136"
```

and use it for hotbar menu empty cells

hotbar menu

**Review by aleksfront — APPROVED** [review]

**Pull request merged**